plugins/trace_api_plugin/include/eosio/trace_api/compressed_file.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <ios>
+#include <fc/io/cfile.hpp>
+
+namespace eosio::trace_api {
+
+   class compressed_file_datastream;
+   struct compressed_file_impl;
+   /**
+    * wrapper for read-only access to a compressed file.
+    * compressed files support seeking though the efficiency of random access is lower than
+    * an uncompressed file in trade for smaller on-disk resource usage
```

Comment:
I think this sentence could use some work.

Reply:
"Compressed files support seeking. The efficiency of random access is lower than an uncompressed file. The trade off is smaller on-disk resource usage."